### ISSUE TITLE
Add editor-specific exposure controls within the 3D viewport

### ISSUE THREAD

**Issue by lander-vr** [godotengine/godot-proposals#9532]
### Describe the project you are working on

A high-fidelity 3D environment.

### Describe the problem or limitation you are having in your project

Working in editor with physical light units in high dynamic range lighting currently is quite cumbersome. Exposure is constantly fluctuating as you would expect it to in a game, but for level editing and lighting purposes this quickly becomes tedious to work with.

There is no way to easily or quickly manage in-editor exposure. This makes it even more difficult and tedious to correctly set up high dynamic range lighting, particularly in dynamic lighting conditions.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

An exposure-lock button accessible in the 3D viewport that switches between a selected camera-attributes .tres, and an exposure locked at an easily adjustable exposure value.

This would make it drastically easier to create lighting setups that match [real world exposure values](https://en.wikipedia.org/wiki/Exposure_value#Tabulated_exposure_values), and set up lights so they behave correctly in varying lighting conditions.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

Looking at the current 3D viewport UI, I think a place where this setting could make sense is within this editor view submenu:
![image](https://github.com/godotengine/godot-proposals/assets/31851431/017b527d-a2b3-4d70-8e32-35011e981c55)

or alternatively in this "View" menu:
![image](https://github.com/godotengine/godot-proposals/assets/31851431/80b860d7-c880-409f-a504-468032f64e58)

### If this enhancement will not be used often, can it be worked around with a few lines of script?

This enhancement would be used often.

### Is there a reason why this should be core and not an add-on in the asset library?

It adds support for existing and established lighting workflows, improves workflows around already existing functionality in Godot, and improves the quality of life in the editor.

**Comment by clayjohn** [comment]
What do you think about instead having a "preview camera attributes" button? It could sit alongside the preview sun and preview sky buttons and be tweakable with them as well

![Screenshot from 2024-04-16 13-34-53](https://github.com/godotengine/godot-proposals/assets/16521339/1727283e-dd31-49eb-b1d1-8724a64e800e)

**Comment by clayjohn** [comment]
I see, indeed "preview camera attributes" would make the most sense as something that gets replaced once you have a WorldEnvironment in the scene (similar to the current preview environment).

**Comment by lander-vr** [comment]
Exactly, but I would also argue that a preview camera attributes doesn't solve the problems/limitations I listed in my proposal.

I think a permanent EV100 slider that can easily be toggled on/off would be more appropriate, so a user can conveniently switch between exposure set through camera attributes set up in the environment node (i.e. the exposure that'd be present in-game), and a static EV in editor.

As a point of reference, Unreal Engine has an exposure locking slider and, though simple, it's proved itself to be an incredibly useful tool which Godot definitely misses an equivalent of. Especially since Godot has all the functionalities to create HDR scenes, this would be a nice feature that supports that functionality and would make for a much nicer experience working with HDR lighting. 
Even more so because (as far as I'm aware) there are no exposure debugging views (Heatmap/histogram), and this would allow you to debug unexpected exposure behavior yourself much easier by being able to manually lock it to specific EV100s.
